hal/targets/hal/TARGET_ONSEMI/TARGET_NCS36510/spi_api.c

```diff
@@ -59,24 +59,28 @@ void spi_free(spi_t *obj)
 
 void spi_format(spi_t *obj, int bits, int mode, int slave)
 {
-    if(slave) {
+    if(slave) 
+	{
         /* Slave mode */
         obj->membase->CONTROL.BITS.MODE = SPI_SLAVE_MODE;
-    } else {
+    } 
+	else 
+	{
         /* Master mode */
         obj->membase->CONTROL.BITS.MODE = SPI_MASTER_MODE;
     }
     obj->membase->CONTROL.BITS.WORD_WIDTH = bits >> 0x4; 			/* word width */
     obj->membase->CONTROL.BITS.CPOL 	  = mode >> 0x1; 			/* CPOL  */
     obj->membase->CONTROL.BITS.CPHA 	  = mode & 0x1; 			/* CPHA */
 
-    obj->membase->CONTROL.BITS.ENDIAN 	  = SPI_ENDIAN_LSB_FIRST;	/* Endian */
+    obj->membase->CONTROL.BITS.ENDIAN 	  = SPI_ENDIAN_MSB_FIRST;	/* Endian */
 }
 
 void spi_frequency(spi_t *obj, int hz)
 {
     /* If the frequency is outside the allowable range, set it to the max */
-    if(hz > SPI_FREQ_MAX) {
+    if(hz > SPI_FREQ_MAX) 
+	{
```

Comment:
Could you please revert this change to the bracket? It does not fit with our style guide: https://github.com/ARMmbed/mbed-os/blob/master/docs/COMMITTERS.md#c-coding-rules--coding-guidelines